Documentation/troubleshooting.md

```diff
@@ -16,6 +16,17 @@ Learn more about configuring Visual Studio Code settings [here](https://code.vis
 
 The .NET Install Tool for Extension Authors requires TLS 1.2 to be enabled in order to install .NET. For more information on TLS1.2, see [the documentation](https://docs.microsoft.com/mem/configmgr/core/plan-design/security/enable-tls-1-2-client).
 
+## Manually Installing .NET
+
+If .NET installation is failing or you want to reuse an existing installation of .NET, you can use the `dotnetAcquisitionExtension.existingDotnetPath` setting. .NET can be manually installed from [the .NET website](https://aka.ms/dotnet-core-download). To direct this extension to that installation, update your settings with the version and the path as illustrated below. Note that if the version is marked as `all` then this location will be used for any version of .NET requested.
+
+```json
+    "dotnetAcquisitionExtension.existingDotnetPath": [
+        {"version": "3.1", "path": "C:\\Program Files\\dotnet"},
```

Comment:
It's not clear what the user is pointing to - the folder?  the dotnet.exe file?  Having real-world examples for Mac and PC would be very helpful.